add settings document (#214)
usebruno/bruno-docs: src/pages/get-started/configure/settings.mdx

![general-settings](/screenshots/get-started/config/settings/general-settings.webp)

## Display

In the Display section, you can control the appearance of the Bruno, choosing between Dark, Light, or System mode, based on your preference. Additionally, you can customize the code editor font to suit your style.

![display-settings](/screenshots/get-started/config/settings/display-settings.webp)

## Proxy

You can customize the Proxy settings by selecting On, Off, or System, depending on your needs. For more details, you can refer to the [Proxy guide here](./proxy-config.mdx) to learn more. By default, the Proxy setting is Off in Bruno.

![proxy-settings](/screenshots/get-started/config/settings/proxy-settings.webp)

## License

Bruno offers Ultimate and Pro plans for premium users. You can refer to the [Licensing guide](/license-management/overview.mdx) to learn more about the features and benefits of each plan.

![license-settings](/screenshots/get-started/config/settings/license-settings.webp)

## Keybindings

> The Keybindings section provides details about Bruno’s keyboard shortcuts, allowing you to easily customize and navigate the application.

Reviewer: ```suggestion
The Keybindings section provides details about Bruno’s keyboard shortcuts.
```

We don't currently allow you to customize the keybindings, correct? If so, I've removed that reference.

![license-settings](/screenshots/get-started/config/settings/key-settings.webp)

> **Secrets**: Vault Secrets is available in the Ultimate Edition. To learn more about configuring and managing secrets, refer a [Secret Provider guide](/secrets-management/hashicorp-vault/adding-a-secret-provider).

Reviewer: ```suggestion
## Secrets

There are three primary ways to manage secrets in Bruno, which you can learn about in the [Secret Management Overview](/secrets-management/overview). 

This setting controls the integration with a Vault for secret management, which is available in Bruno Ultimate Edition. 
```

I've removed the specific reference to Hashicorp Vault since we'll have to restructure that part of the docs once AWS and Azure vaults are released within the next few weeks.

> **Support and About**: Access troubleshooting resources, contact support and view application details and enable auto update feature.

Reviewer: ```suggestion
## Support

Find helpful resources and ways to engage with the Bruno community and team. 

## About

Information related to your installation of Bruno and the ability to set preferences for auto-updates. 
```
